core/tx_pool.go

```diff
@@ -667,6 +668,13 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 		log.Debug("validateTx gas less than intrinsic gas", "tx.Gas", tx.Gas(), "intrinsic Gas", intrGas)
 		return ErrIntrinsicGas
 	}
+
+	gasPriceMinimum, _ := gpm.GetGasPriceMinimum(tx.GasCurrency(), nil, nil)
```

Comment:
any reason to ignore this error?

Reply:
I agree, we should never silently ignore errors. At the least, we should log them.

Reply:
Great point.  I also agree that we shouldn't ignore errors from `GetGasPriceMinimum`.  However, if you look at the `GetGasPriceMinimum` implementation, it will never return a non nil err.

I think that there is a broader error handling issue in `GetGasPriceMinimum`, and I'm going to create a separate issue to address that.  I'm going to merge this PR and we can address the GetGasPriceMinimum ignoring error separately.

@tkporter @ashishb - Let me know if you guys have an issue with that.

Reply:
I actually went ahead and added error handling in txpool for get price minimum.  For now, that code path will never get executed, but one the error handling for `GetGasPriceMinimum` is fixed, then that code will become relevant.

Reply:
Cool, sounds good to me